lib/Notes/components/ReferredRecord/tests/referredRecord-test.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import {
+  describe,
+  beforeEach,
+  it,
+} from '@bigtest/mocha';
+import { expect } from 'chai';
+
+import ReferredRecord from '../ReferredRecord';
+import ReferredRecordInteractor from './interactor';
+import {
+  mount,
+  setupApplication,
+} from '../../../../../tests/helpers';
+
+
+describe('ReferredRecord', () => {
+  const referredRecord = new ReferredRecordInteractor();
+  const referredEntityData = {
+    name: 'Test Name',
+    type: 'Type',
+    id: '1',
+  };
+  const entityTypeTranslationKeys = { [referredEntityData.type]: 'Test' };
+
+  describe('rendering referredRecord component', () => {
+    setupApplication();
```

Comment:
Do we need separate `setupApplication` calls for each `describe` here?

Reply:
No, we don't. Thanks for noticing.